gdal/frmts/grass/GNUmakefile

```diff
@@ -7,7 +7,7 @@ include ../../GDALmake.opt
 ifeq ($(GRASS_SETTING),libgrass)
 OBJ	=	grassdataset.o
 else
-OBJ	=	grass57dataset.o
+OBJ	=	grass.o
```

Comment:
the old grassdataset.cpp (was this for GRASS 4 ?) should be removed too, as well as traces of it like 2 lines above

Reply:
> Have you tried running autotest/gdrivers/grass.py locally ? There isn't much in it. I don't currently have a GRASS setup. And our CI doesn't build with the GRASS driver

Yes, it runs fine:

```
23:37 $ pytest gdrivers/grass.py
Test session starts (platform: linux, Python 3.8.5, pytest 4.6.11, pytest-sugar 0.9.4)
rootdir: /home/mneteler/software/gdal/autotest, inifile: pytest.ini
plugins: sugar-0.9.4, env-0.6.2
collecting ... 
 gdrivers/grass.py ✓✓                                                                                                                                        100% 

Results (0.29s):
       2 passed

```